obal/data/playbooks/nightly.yml

```diff
@@ -2,9 +2,38 @@
 - hosts: packages
   serial: 1
   gather_facts: no
-  roles:
-    - build_package
   vars:
     releasers:
       - "{{ nightly_releaser }}"
-    build_package_tito_releaser_args: "{{ nightly_package_tito_releaser_args }}"
+    build_package_tito_releaser_args: "{{ nightly_package_tito_releaser_args | default('') }}"
+  tasks:
+    - set_fact:
+        nightly_noclean: false
+      when: nightly_noclean is not defined
+
+    - name: copy source file
+      copy:
+        src: "{{ item }}"
+        dest: "{{ inventory_dir }}/{{ package_base_dir }}/{{ inventory_hostname }}/"
+      with_items: "{{ nightly_sourcefiles.split(',') }}"
```

Comment:
`nightly_sourcefiles` is then to be defined on the CLI? tests are failing as that's currently undefined and I don't completely see how this functionality is to be used.